addons/im_livechat/static/src/embed/core/autopopup_service.js

```diff
@@ -58,9 +58,7 @@ export class AutopopupService {
      */
     async shouldOpenChatWindow() {
         const thread = await this.threadService.getLivechatThread();
-        return this.storeService.chatWindows.every(
-            (chatWindow) => chatWindow.thread.localId !== thread?.localId
-        );
+        return this.storeService.chatWindows.every((chatWindow) => !chatWindow.thread?.eq(thread));
```

Comment:
not related to this diff, but I feel `every` + `!` is more complicated to read than `!` + `some`
(especially when condition inside is `eq` but we actually take the negation)